visibility-sensor.js

```diff
@@ -22,6 +22,13 @@ module.exports = React.createClass({
     };
   },
 
+  getInitialState: function () {
+    return {
+      isVisible: true,
```

Comment:
should `isVisible` be `null` by default? Otherwise the first `onChange` may not be fired.